utils.groovy

```diff
@@ -73,6 +73,13 @@ def getFlower() {
   return readFile(file: 'version/RELEASE').replaceAll(" ", "-").toLowerCase().trim()
 }
 
+def getVersionFilename(String version) {
+  return sh(
+    script: "echo ${version} | sed -e 's/[^a-zA-Z0-9-]/-/g' | sed -e 's/--*/-/g'",
+    returnStdout: true
+  ).trim()
```

Comment:
Opportunity to use `replaceAll` instead of running this through sh?
```suggestion
  return version.replaceAll('[^a-zA-Z0-9-]', '-').replaceAll('--*', '-').trim()
```

Reply:
Much better